docs/aerodromes/classd/Moorabbin.md

```diff
@@ -81,13 +116,88 @@ Aircraft who will transit Class G airspace on climb into CTA will be **cleared t
 
 !!! example
     <span class="hotline">**MDS** -> **MB ADC**</span>: "To the west, PLE, for the NDB-A"  
     <span class="hotline">**MB ADC** -> **MDS**</span>: "PLE, NDB-A"
 
 !!! tip
-    Remember that IFR aircraft are only separated from other IFR or SVFR aircraft in class D. Use other separation methods, such as holding a departure on the ground, if separation is required with these aircraft.
\ No newline at end of file
+    Remember that IFR aircraft are only separated from other IFR or SVFR aircraft in class D. Use other separation methods, such as holding a departure on the ground, if separation is required with these aircraft.
+
+## Helicopters
+
+### Circuits
+
+#### Day VFR Circuits
+Helicopter circuits are conducted from designated areas based on the active runway:
+
+<figure markdown>
+![Eastern Grass and Western Triangle](img/mbhelicircuitareas.png){ width="700" }
+<figcaption>Eastern Grass and Western Triangle</figcaption>
+</figure>
+
+- **RWY 17/35**: Circuits are conducted from the **Eastern Grass**.
+- **RWY 13/31**: Circuits are conducted from the **Western Triangle**.
+- Circuit operations are **parallel to the duty runway** and inside the fixed-wing circuit.
+
+**Radio Procedure**:
+- Pilots will report “AIRBORNE” before each circuit, or if departing additionally they will include their intentions. Aircraft are expected to remain under `A001` until you acknowledge.
+- Reply with simply the aircraft callsign for a leg of the circuits, or departure instructions. Hold the aircraft if they will conflict with other traffic.
+- On completion of circuit training, the pilot will nominate their landing HLS during the airborne call and report downwind before transitioning to the arrival procedure.
+
+!!! example
+    **RJB**:: "MB Gnd, RJB, At the southern apron, with information november, request circuits"
+    **MB SMC**: "RJB, cleared for circuit, Eastern Grass, report airborne"      
+    **RJB**: "Cleared for circuits, Eastern Grass, RJB"  
+    *(Before departure)*  
+    **RJB**: "Moorabbin Tower, RJB, airborne"  
+    **MB ADC**: "RJB"
+
+#### Night Circuits
+Night operations must comply with fixed wing circuits and altitudes.
+
+### Departures
+
+Day VFR helicopter departures must remain at `A007`. Helicopters must nominate a aiming point or HLS on first contact with MB SMC.
+
+1. **Aiming Points**:
+   - Direct helicopters departing from the **Southern Aiming Point** to hold short of **TWY A**.
+   - Direct helicopters departing from the **Northern Aiming Point** to hold short of **TWY G**.
+
+2. **Departure Procedures**:
+    - Ensure departures remain clear of duty runways and provide instructions to:
+        - **Pass east of the tower**.
+        - **Track**:
+            - For **northbound departures**: Track north until clear of the tower.
+            - For **southbound departures**: Track south until clear of airport buildings.
```

Comment:
Looks great, could we possibly get an example so controllers can see how this works in practice?